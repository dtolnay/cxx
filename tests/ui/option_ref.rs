#[cxx::bridge]
mod ffi {
    extern "Rust" {
        fn f(_o: &Option<&u8>);
    }
}

fn f(_o: &Option<&u8>) {}

fn main() {}
