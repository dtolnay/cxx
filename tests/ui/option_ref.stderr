error: unsupported reference type
 --> tests/ui/option_ref.rs:4:18
  |
4 |         fn f(_o: &Option<&u8>);
  |                  ^^^^^^^^^^^^
